mifosng-android/src/main/java/com/mifos/mifosxdroid/passcode/PassCodeActivity.java

```diff
@@ -84,15 +83,17 @@ public void savePassCode() {
         }
     }
 
-    @OnClick(R.id.btn_login)
-    public void loginUsingPassCode() {
+    @Override
+    public void passCodeEntered(String passcode) {
 
         if (!isInternetAvailable()) {
+            passCodeView.clearPasscodeField();
```

Comment:
Why are you checking internet connection, we are saving the passcode locally and our app some offline feature too so we don't need to check internet connection.